CONTRIBUTING.rst

```diff
@@ -0,0 +1,56 @@
+=============================
+Contribute to this repository
+=============================
+
+Create Issues
+=============
+
+* If you find something missing or something is wrong in this manual,
+  you are welcome to write an issue describing the problem.
+
+Make changes (create pull requests)
+===================================
+
+You cannot make changes to the documentation in this repository directly. This ViewHelper
+reference is automatically generated from comments in the ViewHelper code source files (PHP).
+
+So, in order to make changes, you will need to make the changes in the repository where the
+source code is kept using the contribution workflow for that repository. And you will need
+to edit the comments in the PHP comments using a basic reST syntax.
```

Comment:
Poorly this is not 100% valid. typo3fluid/fluid is using Markdown in the future, see https://github.com/TYPO3/Fluid/pull/436

I would propose to remove this information here, but add the format to use in the section below for each repository.

Reply:
What do you mean by 

> "add the format to use in the section below for each repository"? 

I will push another commit.